sapl-spring-data-r2dbc/src/main/java/io/sapl/springdatar2dbc/sapl/queries/enforcement/R2dbcAnnotationQueryManipulationEnforcementPoint.java

```diff
@@ -119,7 +120,7 @@ public Function<AuthorizationDecision, Flux<T>> enforceDecision() {
      */
     @SneakyThrows
     @SuppressWarnings("unchecked")
```

Comment:
will not repeat: justify all @SuppressWarnings

Reply:
I don't know how to get rid of these unchecked warnings. 

In this case here (also in the other enforcementPoints) I check which is the return class of the respective method. It can only be Mono OR Flux, Spring's Reactive Repository doesn't allow anything else. If I exclude one, it can only be the other.

This means that it is not an unchecked cast for me, but for the compiler of course. Unfortunately, I can't get it solved.